google/cloud/internal/throw_delegate.h

```diff
@@ -27,8 +27,9 @@ namespace internal {
  * @name Delete exception raising to hidden functions.
```

Comment:
I probably wrote that comment and it makes no sense to me.  Can you change it?

```C++
/**
 * @name Delegate exception raising to out of line functions.
...
```

Reply:
Done.